src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -36,18 +36,19 @@ public function __construct(
         $this->entityManager = $entityManager;
         $this->serializer = $serializer;
         // As `settings` is just an array for now, I think we have to inject Container to do this.
-        $this->webhookSecret = $container->get('settings')['stripe']['webhookSecret'];
+        $this->webhookSecret = (string) $container->get('settings')['stripe']['webhookSecret'];
```

Comment:
Why do we need this cast? Tests should have a value set and it should be a string already. If an env is accidentally set up with no secret, the endpoint should probably crash!

Reply:
@NoelLH CircleCI was throwing this yesterday `TypeError: Typed property MatchBot\Application\Actions\Hooks\StripeUpdate::$webhookSecret must be string, bool used` but on my local it was fine without it being casted to `String`

Reply:
@tbg-jasonfung that suggests you are relying on a var definition which is in your local `.env` but not set up in the test for clean container runs. Have a look at how other vars are overridden in `phpunit.xml[.dist]`

Reply:
@NoelLH Cool thanks, this makes sense, i've removed the casting + updated the xml file